packages/components/data-table-manager/README.md

```diff
@@ -61,10 +66,54 @@ const Example = () => (
 export default Example;
 ```
 
+### Detached
+
+As mentioned earlier, the default behaviour is for the triggering element to be placed above the `<DataTable>` component (top-right corner), but there can be use cases where it would be needed to place that triggering element in a different place in the page. This is also possible but requires the usage of one more component (`DataTableManagerProvider`) in order to share the manager state between the manager panels and the `DataTable` component.
```

Comment:
nit: maybe this reads better:
```suggestion
As mentioned earlier, the default behaviour places the triggering element above the `<DataTable>` component (top-right corner), but there may be use cases where the triggering element needs to be positioned differently on the page. This is also possible but requires the usage of one more component (`DataTableManagerProvider`) in order to share the manager state between the manager panels and the `DataTable` component.
```

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2807/commits/37d6c7c20176cd563a6ad743408f2df3a22a752f

Thanks!